edge/pkg/edged/kubeclientbridge/typed/core/v1/core_client_bridge.go

```diff
@@ -0,0 +1,30 @@
+package v1
```

Comment:
add copyright for the file

Reply:
> add copyright for the file

Already done, and alse with other files.